Template/config/mailgun_id_mapping.php

```diff
@@ -0,0 +1,12 @@
+<h3><img src="<?= $this->url->dir() ?>plugins/Mailgun/mailgun-icon.png"/>&nbsp;Mailgun Unknown ID mapping</h3>
```

Comment:
This text should be translatable.

Reply:
I have added the translation tag......do I need to provide the translations?